web/src/lib/connectors/connectors.ts

```diff
@@ -227,11 +227,29 @@ Selecting the "Index Recursively" checkbox will index the single page's children
     values: [
       {
         type: "text",
-        query: "Enter the wiki page URL:",
-        label: "Wiki Page URL",
-        name: "wiki_page_url",
+        query: "Enter the wiki base URL:",
```

Comment:
We should probably update the description above, since it's for the URL-based model.